src/Giveaway.js

```diff
@@ -105,7 +105,11 @@ class Giveaway extends EventEmitter {
     }
 
     get exemptMembersFunction() {
-        return this.options.exemptMembers ? (this.options.exemptMembers.toString().includes('function anonymous') ? this.options.exemptMembers : eval(this.options.exemptMembers)) : null;
+        return this.options.exemptMembers
+            ? (typeof this.options.exemptMembers === 'string' && this.options.exemptMembers.includes('function anonymous'))
+                ? eval(`(${this.options.exemptMembers})`)
```

Comment:
```js
console.log(eval("function anonymus() { return true }")) // undefined 
console.log(eval("(function anonymus() { return true })")) // function anonymus() { return true } // typeof === function
```
`bonusEntries` does not need this because there the whole array gets serialized and not the function